### PR TITLE
Updates the cron schedule docs with the correct DoW

### PR DESCRIPTION
**Pull request by JoshVanL**
Updates to correct value. Reported from https://github.com/dapr/dapr/pull/8112

### CODE REVIEW

**Comment by JoshVanL** [comment]
Credit @WhitWaldo

File: api/job.pb.go [hunk 1]
```diff
@@ -5,7 +5,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.33.0
-// 	protoc        v4.24.4
+// 	protoc        v4.25.4
```
Comment (cicoyle) [inline api/job.pb.go:8]: can this diff be removed? [We arent using this version in master currently](https://github.com/diagridio/go-etcd-cron/blob/master/Makefile#L39)
```suggestion
// 	protoc        v4.24.4
```

File: api/counter.pb.go [hunk 2]
```diff
@@ -5,7 +5,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.33.0
-// 	protoc        v4.24.4
+// 	protoc        v4.25.4
```
Comment (cicoyle) [inline api/counter.pb.go:8]: same here
```suggestion
// 	protoc        v4.24.4
```

File: api/trigger.pb.go [hunk 3]
```diff
@@ -5,7 +5,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.33.0
-// 	protoc        v4.24.4
+// 	protoc        v4.25.4
```
Comment (cicoyle) [inline api/trigger.pb.go:8]: same here
```suggestion
// 	protoc        v4.24.4
```

File: api/job.pb.go (hunk 1 above)
Comment (cicoyle) [reply api/job.pb.go:8]: we are updating it in the other PR in this repo and in dapr so Im fine with this I suppose

**Review by cicoyle — APPROVED** [review]

**Pull request merged**